benchmark/buffers/buffer-compare-instance-method.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+var common = require('../common.js');
+
+var bench = common.createBenchmark(main, {
+  size: [16, 512, 1024, 4096, 16386],
+  millions: [1]
+});
+
+function main(conf) {
+  var iter = (conf.millions >>> 0) * 1e6;
+  var size = (conf.size >>> 0);
+  var b0 = new Buffer(size).fill('a');
+  var b1 = new Buffer(size).fill('a');
+
+  b1[size - 1] = 'b'.charCodeAt(0);
+
```

Comment:
Please insert code to optimize `b0.compare` before starting the benchmark. Take a look at some of the other (e.g. `path`) benchmarks to see how to do this.